app/components/channel_loader.js

```diff
@@ -103,3 +126,15 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
         }
     };
 });
+
+function mapStateToProps(state, ownProps) {
+    const {deviceWidth} = state.device.dimension;
+    return {
+        ...ownProps,
```

Comment:
Do we need this here? I'm pretty sure the props get passed to the component regardless of if they are included here.

Reply:
Every component that we have does this, so i’m inclined to leave it here

Reply:
If they do get passed in automatically, that'll save us a bit of work when creating new containers. It'll also save us from having to figure out if `...ownProps` should become before or after the rest of the props since we do it both ways

Reply:
we can figure that later, perhaps when building a new component and if that's the case we can go and remove it from everywhere